### PR TITLE
1289 persistence db2

### PR DESCRIPTION
**Pull request by ManasiGandhi**

### CODE REVIEW

File: modules/reference/pages/feature/sessionDatabase-1.0/examples.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+== Examples
+
+=== Minimum configuration to persist session data to a database
+
+The following example shows the minimum configuration for database persistence:
+
+[source, java]
+----
+<fileset id="DerbyFiles" includes="*.jar" dir="server1/derby/client"/>
```
Comment (dmuelle) [inline modules/reference/pages/feature/sessionDatabase-1.0/examples.adoc]: here and line 14, need to remove "server1" per convo here: https://github.com/OpenLiberty/docs/issues/2867#issuecomment-720805100

suggest:
`dir="derby/client"/>`

File: modules/reference/pages/feature/sessionDatabase-1.0/examples.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<library id="DerbyLib" filesetRef="DerbyFiles"/>
+<jdbcDriver id="DerbyDriver" libraryRef="DerbyLib"/>
+<dataSource id="SessionDS" jdbcDriverRef="DerbyDriver">
+    <properties.derby.client user="user1" password="password1"
+                             databaseName="server1/databases/SessionDB"
```
Comment (dmuelle) [inline modules/reference/pages/feature/sessionDatabase-1.0/examples.adoc]: here recommend

`databaseName="SessionDB"`

**Review by dmuelle — CHANGES_REQUESTED** [review]
minor changes to remove `server1`

File: modules/reference/pages/feature/sessionDatabase-1.0/examples.adoc (hunk 2 above)
Comment (ManasiGandhi) [reply modules/reference/pages/feature/sessionDatabase-1.0/examples.adoc]: Thanks. I made the updates.

**Review by dmuelle — APPROVED** [review]

**Pull request merged**